src/content/peerconnection/trickle-ice/js/main.js

```diff
@@ -247,6 +245,16 @@ function iceCallback(event) {
       return;
     }
     const {candidate} = event;
+    const stats = await pc.getStats();
```

Comment:
nit: move "await pc.getStats()" to inside the condition to avoid unnecessarily polling getStats